persist-cli/src/main/java/io/ballerina/persist/nodegenerator/BalSyntaxTreeGenerator.java

```diff
@@ -215,9 +221,22 @@ public static EntityMetaData getEntityRecord(Path filePath)
                                 .equals(KEYWORD_TABLE_NAME)) {
                             tableName = ((BasicLiteralNode) valueNode).literalToken().text()
                                     .replaceAll(DOUBLE_QUOTE, BalFileConstants.EMPTY_STRING);
+                        } else if (((SpecificFieldNode) fieldNode).fieldName().toString().trim().
+                                equals(UNIQUE_CONSTRAINTS)) {
+                            uniqueConstraints = new ArrayList<>();
+                            for (Node node : ((ListConstructorExpressionNode) valueNode).expressions()) {
+                                List<String> uniqueConstraint = new ArrayList<>();
+                                for (Node value : ((ListConstructorExpressionNode) node).expressions()) {
+                                    uniqueConstraint.add(value.toSourceCode());
+                                }
+                                uniqueConstraints.add(uniqueConstraint);
+                            }
                         }
                     }
-                    entityArray.add(index, new Entity(getArray(keys), tableName));
+                    if (tableName == null) {
```

Comment:
This code segment is repeated in Entity.java https://github.com/ballerina-platform/persist-tools/pull/56/files#diff-8e5ff4f02e822d33f8c6c9ef6b18e82f225eb70d630ab23b18cc908608735650L48:~:text=%7D-,public,-String%20getTableName()%20%7B

Reply:
I got an empty table name when missing the table name in annotation. so I added this to get this.

Reply:
I got an empty table name when missing the table name in the annotation. So I added this to get this.